ansible-st2/roles/st2/defaults/main.yml

```diff
@@ -1,28 +0,0 @@
----
-# Required memory for st2 installation in MB
-st2_required_memory: 2000
-# MongoDB requires at least 3379MB available in /var/lib/mongo/journal
-st2_required_space: 3500
-
-# 'stable', 'unstable' to get latest version or numeric like '0.12.1'
-st2_version: stable
-# 'current' to get latest revision or numberic like '6'
-st2_revision: current
-
-st2_system_user: stanley
-st2_ssh_key_file: /home/{{ st2_system_user }}/.ssh/{{ st2_system_user }}_rsa
-
-st2_packages:
-  - st2common
-  - st2actions
-  - st2api
-  - st2auth
-  - st2client
-  - st2debug
-  - st2reactor
-
-# Number of action runners to register. Defaults to number of vCPUs, but not less than 2
-st2_action_runners: "{{ [ansible_processor_vcpus, 2] | max }}"
```

Comment:
Maybe we want more than 2 action runners installed on our systems? Don't know how light-weight they are.